content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Optimizing DVC Remotes
```

Comment:
probably needs a more interesting title

Reply:
from the intro think this post is more about "Optimization improvements in DVC 1.0"